File: app/src/main/java/org/miaowo/miaowo/service/WebService.java
package org.miaowo.miaowo.service;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

public class WebService extends Service {

    public WebService() {
        // TODO: 用于接收服务器新消息
    }

    @Override
    public IBinder onBind(Intent intent) {
        throw new UnsupportedOperationException("Not yet implemented");
    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        return super.onStartCommand(intent, flags, startId);
    }

    @Override
    public void onDestroy() {
        super.onDestroy();
    }
}

File: app/src/main/java/org/miaowo/miaowo/service/WebService.kt
package org.miaowo.miaowo.service

import android.app.Service
import android.content.Intent
import android.os.Binder
import android.os.IBinder

class WebService : Service() {

    override fun onBind(intent: Intent): IBinder? {
        return WebBinder()
    }

    override fun onDestroy() {
        super.onDestroy()
    }

    class WebBinder: Binder() {
        // TODO: 用于接收服务器新消息
    }
}
